### PR TITLE
docs(docker): guide to purge testing database

### PR DESCRIPTION
**Pull request by tai271828**
Add guide regarding how to purge the testing DB without rebooting your system.

### CODE REVIEW

**Comment by tai271828** [comment]
@EagleC0318 @yanghaochang104 I think this is what we talked about ~1 weeks ago.

**Comment by tai271828** [comment]
This pull request relates to https://github.com/OH-SHOWN/ohshown-backend/issues/54

**Comment by tai271828** [comment]
If anyone tries the method and it's ok for you. Please let me know. I will then upstream this README update to disfactor repo as well : )

**Comment by EagleC0318** [comment]
@tai271828 if I run this purge command and then start the test docker env again, would the default test data be still there?
If so, what's the data I purged? The data I added manually on the last test?

File: backend/docs/SETUP_COMPOSE.md [hunk 1]
```diff
@@ -59,6 +59,13 @@ docker-compose exec web python manage.py createsuperuser
 docker-compose -f docker-compose.dev.yml down
 ```
 
+If you want to also purge your testing database, stop the containers and remove the corresponding volumes in the host
+
+```bash
+docker-compose -f docker-compose.dev.yml down --rmi all --volumes
```
Comment (EagleC0318) [inline backend/docs/SETUP_COMPOSE.md:65]: Wondering why we need `--rmi all` here to purge the database?

Comment (EagleC0318) [reply backend/docs/SETUP_COMPOSE.md:65]: Hmm these two commands haven't purged the ohshown event I added manually. Still can see it after ran `docker-compose -f docker-compose.dev.yml up -d` again.

File: backend/docs/SETUP_COMPOSE.md [hunk 2]
```diff
@@ -59,6 +59,13 @@ docker-compose exec web python manage.py createsuperuser
...
+rm -rf /tmp/disfactory_d*
```
Comment (tai271828) [inline backend/docs/SETUP_COMPOSE.md:66]: @EagleC0318 did you apply this line as well? It needs both.

Comment (EagleC0318) [reply backend/docs/SETUP_COMPOSE.md:66]: @tai271828 I think so. In fact, I feel there're actually no data under `/tmp/disfactory_d*` in my local, but the new ohshown event still there when start the container again.

Comment (EagleC0318) [reply backend/docs/SETUP_COMPOSE.md:66]: Could it be cached somewhere else?

Comment (tai271828) [reply backend/docs/SETUP_COMPOSE.md:66]: How do you define your `DISFACTORY_PGDATA_PATH` and `DISFACTORY_PGDATA_BACKUP_PATH` in `.env`?

See here for reference:
https://github.com/OH-SHOWN/ohshown-backend/blob/5fcf3370297c761b61f3c6058b03e73586701ca5/backend/docker-compose.dev.yml#L35

Comment (EagleC0318) [reply backend/docs/SETUP_COMPOSE.md:66]: I think it's the same, didn't modify the docker/env config in my local.

Comment (tai271828) [reply backend/docs/SETUP_COMPOSE.md:66]: Then I have no idea XD We will see when we need it.

Comment (EagleC0318) [reply backend/docs/SETUP_COMPOSE.md:66]: Yeah, it's fine for me if you want to check in this document change first.

**Pull request merged**